hydra/_internal/config_loader_impl.py

```diff
@@ -206,6 +206,13 @@ def _err() -> None:
             new_csp.append(provider=provider, path=sp)
         new_csp.append("schema", "structured://")
         repo.initialize_sources(new_csp)
+        for source in repo.get_sources():
+            if source.provider == provider:
+                if not source.available():
+                    warnings.warn(
+                        category=UserWarning,
+                        message=f"hydra.searchpath={searchpath} is not available.",
```

Comment:
This seems wrong.
you are checking source and you are printing about searchpath.

Reply:
per offline chats, the source here is filtered to be for `searchpath` which is why the message says searchpath.

since we want the warnings for all the sources, I will move this check elsewhere.